### DISCUSSION TITLE
Is there any argument like pip's index-url?

### DISCUSSION THREAD

**Discussion by miss85246** [pdm-project/pdm#1032] [Q&A] [answered]
As we know, we can install packages from different mirror source with index-url:

```shell
pip install  sanic-dantic -i https://pypi.tuna.tsinghua.edu.cn/simple
```
I know I can set pip's global config to change index-url.  
But you know, different mirror source has different sync rules. for example:
aliyun pip mirror will sync pacakge every day. It's very fast
tsinghua synchronized it every 5 minutes.but a little slow

the aliyun is my primary mirror source, when I released a new pacakge in pipy, I need use tsinghua mirror source to install it.

Is there anyway can let me install pacakge only once from tsinghua mirror source ?

**Comment by songololo** [comment]
Another with the same question, for example, how would I add this to `pyproject.toml` for use with PDM?
```bash
pip3 install -i https://pypi.anaconda.org/numba/label/wheels_experimental_m1/simple numba
```

**Reply by Hnasar** [reply]
You can set `PIP_INDEX_URL` environment variable, and since PDM uses pip under the hood, it will respect it.

**Reply by baggiponte** [reply]
So `unearth` respects `PIP_*` env variables? @frostming

**Reply by frostming** [reply]
No

**Comment by frostming** [comment, marked as answer]
Let me know PDM's [source settings](https://pdm.fming.dev/pyproject/tool-pdm/#specify-other-sources-for-finding-packages) solves your problem or why not.

**Reply by songololo** [reply]
Yes, that works, thanks.
